src/main/webapp/js/user-page-loader.js

```diff
@@ -32,16 +32,19 @@ function setPageTitle() {
 /**
  * Shows the message form if the user is logged in and viewing their own page.
  */
-function showMessageFormIfViewingSelf() {
-  document.getElementById('about-me-form').classList.remove('hidden');
+
+ //Show the form as long as the user is logged in, even if they aren't viewing their own page
+ //Rename function to accurately reflect what it does
+function showMessageFormIfLoggedIn() {
+  document.getElementById('about-me-form').classList.remove('hidden'); //teammate doing About Me project added this line
```

Comment:
Thanks for correctly handling this line (it wasn't in the sample code but you correctly saw that a teammate added this line, so you kept it in). But please remove the comment (that is, change the line back to how it was) - who added which line can be seen from revision history, and changing lines unnecessarily makes it slightly harder both to resolve merge conflicts and, when looking through history, to see who did what.